spec/support/form_helpers.rb

```diff
@@ -332,6 +339,9 @@ def fill_in_activity_form(
     end
     expect(page).to have_content oda_eligibility
     expect(page).to have_content oda_eligibility_lead if level == "project" || level == "third_party_project"
+    if (level == "project" || level == "third_party_project") && parent.is_newton_funded?
```

Comment:
I'm not sure this will work for projects, because (as I understand it) their parent is the fund itself, so `parent.is_newton_funded?` will return false, even if the direct parent is the Newton fund.

Reply:
Just when I think I understand everything! 😆